docs/readmes/proposals/p017_mme_migrate_to_c++.md

```diff
@@ -0,0 +1,134 @@
+---
+id: p017_mme_migrate_to_c++
+title: C++ Migration
+hide_title: true
+---
+
+
+# Proposal: C++ Migration
+
+Author(s): [Author Name, Co-Author Name]
```

Comment:
Should this have your name? :D

Reply:
Many items have been suggested by Ulas and Scott, I will put my name along their name also.